benchmarks/java/src/main/java/com/google/protobuf/ProtoBench.java

```diff
@@ -66,17 +70,26 @@ public static void main(String[] args) {
       System.exit(1);
     }
     boolean success = true;
+    int endIndex = args.length;
     for (int i = 0; i < args.length; i++) {
-      success &= runTest(args[i]);
+      if (args[i].equals("--")) {
```

Comment:
What is this used for?

Reply:
This is for user to add caliper options. All the strings after "--" will be considered to be caliper options.